python/sdk/prelude_sdk/controllers/http_controller.py

```diff
@@ -20,3 +21,43 @@ def __init__(self):
 
         self._session.mount("http://", HTTPAdapter(max_retries=retry))
         self._session.mount("https://", HTTPAdapter(max_retries=retry))
+
+    def get(self, url, retry=False, **kwargs):
```

Comment:
nit: maybe personal pref, but I read this as "do_retry" and expected it to default to True and have it equal false on line 32